lib/brexit_checker/actions.yaml

```diff
@@ -424,11 +425,12 @@ actions:
   title: Check if you need to get your EU professional qualifications recognised in
     the UK
   consequence: You will not be able to work in a regulated profession if your qualification
-    from the EU, Iceland, Liechtenstein, Norway or Switzerland is not recognised in
+    from the EU, Switzerland, Norway, Iceland or Liechtenstein is not recognised in
     the UK.
   lead_time: Do it as soon as possible
   guidance_prompt: More information
-  guidance_link_text: Get your EEA qualification recognised in the UK after Brexit
+  guidance_link_text: Get your qualification from the EU, Switzerland,Norway, Iceland
```

Comment:
Missing space `Switzerland,Norway`

Reply:
! Great catch !
Will go change in the spreadsheet too